set.mm

```diff
@@ -526759,6 +527034,316 @@ can be drawn starting from two points (we take ` 0 ` and ` 1 ` ), and taking
         } ) , { 0 , 1 } ) " _om ) $.
   $}
 
+  ${
+    constrrtll.s $e |- ( ph -> S C_ CC ) $.
+    constrrtll.a $e |- ( ph -> A e. S ) $.
+    constrrtll.b $e |- ( ph -> B e. S ) $.
+    constrrtll.c $e |- ( ph -> C e. S ) $.
+    constrrtll.d $e |- ( ph -> D e. S ) $.
+    constrrtll.x $e |- ( ph -> X e. CC ) $.
+    constrrtll.t $e |- ( ph -> T e. RR ) $.
+    constrrtll.r $e |- ( ph -> R e. RR ) $.
+    constrrtll.1 $e |- ( ph -> X = ( A + ( T x. ( B - A ) ) ) ) $.
+    constrrtll.2 $e |- ( ph -> X = ( C + ( R x. ( D - C ) ) ) ) $.
+    constrrtll.3 $e |- ( ph
+                      -> ( Im ` ( ( * ` ( B - A ) ) x. ( D - C ) ) ) =/= 0 ) $.
+    constrrtll.n $e |- N = ( A
+        + ( ( ( ( ( A - C ) x. ( ( * ` D ) - ( * ` C ) ) )
+        - ( ( ( * ` A ) - ( * ` C ) ) x. ( D - C ) ) )
+      / ( ( ( ( * ` B ) - ( * ` A ) ) x. ( D - C ) )
+        - ( ( B - A ) x. ( ( * ` D ) - ( * ` C ) ) ) ) ) x. ( B - A ) ) ) $.
```

Comment:
hypothesis `X e. CC` can be avoided because `( A + ( T x. ( B - A ) ) ) e. CC` for example

same for constrrtlc1, constrrtlc2

Reply:
I've done the change in https://github.com/metamath/set.mm/pull/4926/commits/da5257bbf5a731a572838fa300ff396b2a8e5134.
However, those theorems are largely lemmas to be used together with `constrtcc`, for which the same cannot be done, so I'm not sure if this is ever going to be used elsewhere.